docs/topics/multiplatform/multiplatform-hierarchy.md

```diff
@@ -104,6 +129,43 @@ kotlin {
 }
 ```
 
+</tab>
+<tab title="Groovy" group-key="groovy">
+
+```groovy
+kotlin {
+    // Enable the default target hierarchy:
+    targetHierarchy.default {
+      
+    }
+
+    jvm()
+    iosArm64()
+    // the rest of the necessary targets...
```

Comment:
```suggestion
    // The rest of the other targets, if needed.
```

Reply:
Maybe set a link to a page with targets?

Reply:
Links in code comments do not resolve, but we have a link to supported targets at the very first paragraph of this page